1-intro/presentation.ipynb

```diff
@@ -72,6 +78,45 @@
     "# Go to www.menti.com and enter code: xxxx xxxx"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": "subslide"
+    },
+    "tags": []
+   },
+   "source": [
+    " - 3.7   Required for minimal version of ``astropy``\n",
```

Comment:
Support for `astropy` 5.0 will be dropped in a couple of weeks, it's not worth mentioning here.

Reply:
Which will be the oldest supported version after this? I see `astropy` 5.1 requires python 3.8.

Reply:
Once 6.0 is released it will be the oldest supported version, and it will require Python 3.9 or newer. Anaconda just updated the `astropy` they provide to 5.3, for that the Python requirement is the same.

Reply:
Ok, then there is not really any point in talking about versions older than 3.9. I will remove those lines.